public/js/test/fixtures/todomvc.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
I removed `todomvc.toggle` because it was very similar to `todomvc.updateOnEnter` and added `todomvc` which is just the app state when the debugger starts.